src/main/java/seedu/address/storage/XmlAdaptedPrescriptionList.java

```diff
@@ -15,26 +15,31 @@
  */
 @XmlRootElement
 public class XmlAdaptedPrescriptionList implements Iterable<XmlAdaptedPrescription> {
-    @XmlElement(required = true)
-    private List<XmlAdaptedPrescription> prescription;
+    @XmlElement(required = true, name = "prescription")
+    private List<XmlAdaptedPrescription> prescriptions;
 
     public XmlAdaptedPrescriptionList() {
-        prescription = new ArrayList<>();
+        prescriptions = new ArrayList<>();
     }
 
-    /** C'tor to wrap a list. */
-    public XmlAdaptedPrescriptionList(List<XmlAdaptedPrescription> prescription) {
-        this.prescription = new ArrayList<>(prescription);
+    /** Defensive copy c'tor. */
+    public XmlAdaptedPrescriptionList(List<XmlAdaptedPrescription> prescriptions) {
+        this.prescriptions = new ArrayList<>(prescriptions);
+    }
+
+    /** Defensive copy c'tor. */
+    public XmlAdaptedPrescriptionList(XmlAdaptedPrescriptionList prescriptions) {
+        this(prescriptions.prescriptions);
```

Comment:
\nit Maybe can consider using a different name for the parameter to avoid duplication in object and field name.